include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_fwd_multiple_abd_xdl_cshuffle.hpp

```diff
@@ -520,6 +520,8 @@ struct DeviceGroupedConvFwdMultipleABD_Xdl_CShuffle
                             I1,
                             I0>;
 
+    // NPerBlock is used for the first and second dim. Second dim is store dimension
+    // (with CBlockTransferScalarPerVector_NWaveNPerXdl scalar per vector).
```

Comment:
Why used for both? Why not MPerBlock x NPerblock. Please elaborate more and be more specific.